build-system/server/app.js

```diff
@@ -65,6 +65,9 @@ app.use('/amp4test', require('./amp4test').app);
 app.use('/analytics', require('./routes/analytics'));
 app.use('/list/', require('./routes/list'));
 app.use('/test', require('./routes/test'));
+if (argv.coverage) {
+  app.use('/coverage', require('istanbul-middleware').createHandler());
```

Comment:
Curious: Does this work with the various `gulp serve` flags like `--esm` or `--compiled` or `--sxg`?

Reply:
`--compiled` yes. As for `--esm` and `--sxg`, I haven't tried those yet, but I suspect they will be slightly more complicated based on some of the related code paths I saw. I'm going to work on `--esm` next week to make sure that's usable and the sourcemaps function properly. As for SxG, I don't have a deep enough understanding to know whether it would have any impact, and whether SxG would be any different from a coverage perspective

Reply:
Based on my understanding, this PR should have a similar effect on `--esm` and `--sxg`. So my guess is that if one of them works with the coverage middleware, both should. We can confirm once the initial code lands.